src/simulator/world.py

```diff
@@ -170,7 +171,18 @@ def visualize(self) -> dict:
         """
         objects_info = dict()
         for oid in self.objects:
-            if self.objects[oid].shape is not None:
-                # TODO: to be replaced with proper handling of compound objects (if cond)
-                objects_info[oid] = self.objects[oid].visualize()
+            # TODO:In higher versions, this should be changed (if the visualize method is changed)
+            objects_info[oid] = self.objects[oid].visualize()
         return objects_info
+    
+    def dump_all_owners_inf(self):
+        # dump visualization info for Owners between objects to the output json file
+        owners_info = {"owners":{}}
+        for Oid in self.objects:
```

Comment:
as stated many times before, names starting with capital letters are reserved for class names.